pkg/resourcegenerator/pdb/pod_disruption_budget.go

```diff
@@ -64,7 +72,7 @@ func determineMinAvailable(replicasAvailable uint) *intstr.IntOrString {
 	if replicasAvailable > 1 {
 		value = intstr.FromString("50%")
 	} else {
-		intstr.FromInt(0)
+		value = intstr.FromInt(0)
```

Comment:
`intstr.FromInt` is deprecated


```suggestion
		value = intstr.FromInt32(0)
```